contracts/src/registry/EnhancedAccessControl.sol

```diff
@@ -36,6 +39,16 @@ abstract contract EnhancedAccessControl is Context, ERC165 {
      */
     bytes32 public constant ROOT_RESOURCE = bytes32(uint256(0xdeadbeef));
     
+    /**
+     * @dev The `SUPERUSER` role.
+     *
+     * This role automatically has all user roles and admin roles.
+     *
+     * Note:
+     * - The admin role bit for this role (128) is unnecessary, but still reserved for clarity purposes.
+     * - The SUPERUSER role is only effective in the `ROOT_RESOURCE`, not in subresources.
+     */
+    uint256 public constant ROLE_SUPERUSER = 1;
```

Comment:
Why introduce this as something distinct from an account that just has all role bits set? It seems to add complexity but no new functionality.